man/systemd.link.xml

```diff
@@ -967,6 +967,17 @@
           <xi:include href="version-info.xml" xpointer="v232"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ReceivePacketSteeringCPUMask=</varname></term>
+        <listitem>
+          <para>Configures Receive Packet Steering (RPS) list of CPUs to which RPS may forward traffic.
+          RPS is disabled when it is zero (the default). Takes a list of CPUs and has the same syntax
+          as a list of CPUs for <varname>CPUAffinity=</varname> option or special "all" value which will
```

Comment:
`CPUAffinity=` supports "numa", but this setting seems to not support that.
I think it is better to copy the explanation for `CPUAffinity=` rather than referencing it here.